README.md

```diff
@@ -155,6 +153,23 @@ See [Systemd socket activation](#systemd-socket-activation)
 The buildkitd daemon listens gRPC API on `/run/buildkit/buildkitd.sock` by default, but you can also use TCP sockets.
 See [Expose BuildKit as a TCP service](#expose-buildkit-as-a-tcp-service).
 
+### Windows Setup
+
+See instructions and notes at [`docs/windows.md`](./docs/windows.md).
+
+### macOS Setup
+
+[Homebrew package](https://formulae.brew.sh/formula/buildkit) (unofficial) is available for macOS.
+```console
+$ brew install buildkit
+```
```

Comment:
It should be explained that the daemon is not included in the brew package

Reply:
(The daemon can be launched as follows)
```
limactl start template://buildkit
export BUILDKIT_HOST="unix://$HOME/.lima/buildkit/sock/buildkitd.sock"
```

Reply:
For this PR, I was just focusing on Windows alone, just happened that I had to re-arrange some things.